tcms/testruns/templates/testruns/get.html

```diff
@@ -117,6 +117,100 @@ <h2 class="card-pf-title" style="text-align: left">
                 <h2 class="card-pf-title">
                     {% trans 'Test executions' %}
                 </h2>
+                <div class="container-fluid">
+                    <div class="row toolbar-pf">
+                        <div class="col-sm-12">
+                            <form class="toolbar-pf-actions">
+                                <div class="form-group toolbar-pf-filter">
+                                    <label class="sr-only" for="filter">Name</label>
+                                    <div class="input-group">
+                                        <div class="input-group-btn">
+                                            <button type="button" class="btn btn-default dropdown-toggle" id="input-filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
+                                            <ul class="dropdown-menu">
+                                                <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                                <li><a href="#">{% trans "Priority" %}</a></li>
+                                                <li><a href="#">{% trans "Category" %}</a></li>
+                                            </ul>
+                                        </div><!-- /input-group-btn -->
+                                        <input type="text" class="form-control" id="filter" placeholder='{% trans "Filter By Name..." %}'>
+                                    </div><!-- /input-group -->
+                                </div>
+                                <div class="form-group">
+                                    <div class="dropdown btn-group">
+                                        <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
+                                        <ul class="dropdown-menu">
+                                            <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                            <li><a href="#">{% trans "Priority" %}</a></li>
+                                            <li><a href="#">Last {% trans "Category" %}</a></li>
+                                        </ul>
+                                    </div>
+                                    <button class="btn btn-link" type="button">
+                                        <span class="fa fa-sort-alpha-asc"></span>
+                                    </button>
+                                </div>
+                                <div class="form-group">
+                                    <button type="button" class="btn btn-default">{% trans "Add case" %}</button>
```

Comment:
This button won't be needed, see the TestCase page - test plan widget. See backlog description as well.